os/android/drmhwctwo.cpp

```diff
@@ -95,7 +95,7 @@ HWC2::Error DrmHwcTwo::Init() {
   displays_.at(HWC_DISPLAY_PRIMARY).Init(disable_explicit_sync_);
 
   // Start the hwc service
-  hwcService_.Start(*this);
+  // hwcService_.Start(*this);
```

Comment:
Add Jira and Test to commit message. Can u add FIXME here with comments what needs to be tested before enabling it again ? Can u create a Jira for this also, so we can track the work to enable this again